nixnet_examples/programmatic_database_api.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import
```

Comment:
`programmatic_database_api.py`

The name feels very awkward to me.  Is there an equivalent C or LV example for what this is trying to accomplish?  What is it named?  What does it look like / do?